AUTHORS.md

```diff
@@ -0,0 +1,8 @@
+# Authors
+
+The following people have made contributions to the tutorials (in alphabetical
+order by last name) and are considered "The Tutorial Developers":
+
+* [Agustina Pesce](https://github.com/aguspesce) - Universidad Nacional de San Juan, Argentina (ORCID: 0000-0002-5538-8845)
+* [Santiago Soler](https://github.com/santisoler) - CONICET, Argentina; Instituto Geofísico Sismológico Volponi, Universidad Nacional de San Juan, Argentina (ORCID: 0000-0001-9202-5317)
```

Comment:
Missing @andieie here 🙁 What's your current info? I'd be happy to add it

Reply:
* [Andrea Balza Morales](https://github.com/andieie) - RWTH Aachen University - Institute for Applied Geophysics and Geothermal Energy  (ORCID: 0000-0001-9202-5317)

Thank you for including me,  I really didn't do much just watched Santi and Agus and gave some feedback. Hopefully, I get more time to do more.

Reply:
👍🏽  added. And we're always happy to have you even if just for a little while 🙂